api/src/org/apache/cloudstack/api/command/admin/vlan/UpdateVlanIpRangeCmd.java

```diff
@@ -0,0 +1,137 @@
+package org.apache.cloudstack.api.command.admin.vlan;
+
+import org.apache.cloudstack.api.APICommand;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.ApiErrorCode;
+import org.apache.cloudstack.api.BaseAsyncCmd;
+import org.apache.cloudstack.api.Parameter;
+import org.apache.cloudstack.api.ServerApiException;
+import org.apache.cloudstack.api.response.VlanIpRangeResponse;
+import org.apache.log4j.Logger;
+
+import com.cloud.dc.Vlan;
+import com.cloud.event.EventTypes;
+import com.cloud.exception.ConcurrentOperationException;
+import com.cloud.exception.ResourceAllocationException;
+import com.cloud.exception.ResourceUnavailableException;
+import com.cloud.user.Account;
+import com.cloud.utils.net.NetUtils;
+
+
+@APICommand(name = "updateVlanIpRange", description = "Updates a VLAN IP range.", responseObject =
+        VlanIpRangeResponse.class, since = "4.13.0",
+        requestHasSensitiveInfo = false, responseHasSensitiveInfo = false)
+public class UpdateVlanIpRangeCmd extends BaseAsyncCmd {
+    public static final Logger s_logger = Logger.getLogger(UpdateVlanIpRangeCmd.class.getName());
+
+    private static final String s_name = "updatevlaniprangeresponse";
```

Comment:
better use the name to put on the Annotation.name= here and add response and to lower in the getCommandName() method. examples of this are plenty (no biggy)

Reply:
done